docs/pages/get_started/designers.js

```diff
@@ -44,17 +44,13 @@ In your first couple of weeks at Pinterest, you will receive an invite from your
           description={`
 **Please note: The following plugins are only available for Pinterest employees.**
 
-[**Gestalt docs for Figma**](https://www.figma.com/community/plugin/977755389228415646/Gestalt-docs-for-Figma-(Beta))
-View the Gestalt documentation without leaving Figma!
-
-[**Pinsert**](https://www.figma.com/community/plugin/763812093925718603/Pinsert)
-Insert Pins, Boards, and Profiles from Pinterest.
+[![promo image for the pinterest design figma plugin](https://i.pinimg.com/originals/2b/4b/b4/2b4bb4f0c0e82ddc8f37917c82738b7d.png)](https://www.figma.com/community/plugin/1215463263194174399)
```

Comment:
can we use here **PrivateLink** instead?

![Screenshot by Dropbox Capture](https://user-images.githubusercontent.com/10593890/228654678-1d71d5ce-2583-485d-8c8a-a4a19fdc82df.png)

Reply:
Doesn't look like that's possible to use w/in `MainSection.Subsection`'s description prop. :(